patches/kolibri-utils-server.diff

```diff
@@ -0,0 +1,42 @@
+--- "kolibri/utils/server.py" 2021-08-11 09:40:50.000000000 -0700
++++ "kolibri/utils/server.py" 2021-08-11 09:40:36.000000000 -0700
+@@ -62,7 +62,7 @@
+ DAEMON_LOG = os.path.join(conf.LOG_ROOT, "daemon.txt")
+
+ # Currently non-configurable until we know how to properly handle this
+-LISTEN_ADDRESS = "0.0.0.0"
++LISTEN_ADDRESS = "127.0.0.1"
+
+
+ class NotRunning(Exception):
+@@ -106,22 +106,22 @@
+         scheduler.start_scheduler()
+
+         # Register the Kolibri zeroconf service so it will be discoverable on the network
+-        from kolibri.core.discovery.utils.network.search import (
+-            register_zeroconf_service,
+-        )
++        # from kolibri.core.discovery.utils.network.search import (
++        #     register_zeroconf_service,
++        # )
```

Comment:
Why leave lines commented instead of just removing? Same in all cases below.

Reply:
> This patch looks small enough. I wonder if we are building the kolibri-app from the main/master branch. If so, we should pick carefully what to merge before the Friday release. We are in freeze, if this is merged we need to build, create a new release candidate image, and test thoughtfully. It's @erikos who has the final word.

The image is built using the last release, so we need to merge, create a tag and upload the `kolibri-windows.zip` to that tag. So it's safe to merge this right now.